webv2/typemap/sqlserver_typemap.go

```diff
@@ -81,6 +81,13 @@ func ToSpannerTypeSQLserver(srcType string, spType string, mods []int64) (ddl.Ty
 			}
 			return ddl.Type{Name: ddl.String, Len: ddl.MaxLength}, nil
 		}
+	case "json":
```

Comment:
json datatype is not present in sql server

Reply:
Updated